.gitignore

```diff
@@ -65,6 +65,7 @@ coverage.xml
 /test/units/cover-html
 /test/integration/targets/*/backup/
 /test/cache/*
+pytestdebug.log
```

Comment:
Leave this out of the PR. If it's something you'd like to include, open a separate PR with an explanation of why it's needed.

Reply:
Argh, I tried to remove this from history and made a mess.

Reply:
Phew, thankfully ORIG_HEAD came to the rescue.